tests/check_convert_util.c

```diff
@@ -64,9 +96,116 @@ START_TEST(test_convert_parse_header){
 }
 END_TEST
 
-START_TEST(test_convert_parse_tlvs){
-	/* TODO */
-	ck_assert(1);
+START_TEST(test_convert_parse_tlvs_generic){
```

Comment:
detail: you might want to add `macro-open START_TEST` in your uncrustify config (and remove all the other specific names from the project you stole this config ;-) )

Reply:
Thanks for the macro-open tip, will add it. I thought I removed everything specific to the other project, but indeed I forgot some stuff at the end, will remove that :-)